src/gui/widgets/Fader.cpp

```diff
@@ -207,22 +207,22 @@ void Fader::wheelEvent (QWheelEvent* ev)
 ///
 void Fader::setPeak(float fPeak, float& targetPeak, float& persistentPeak, QElapsedTimer& lastPeakTimer)
 {
-	fPeak = std::clamp(fPeak, m_fMinPeak, m_fMaxPeak);
-
 	if (targetPeak != fPeak)
 	{
 		targetPeak = fPeak;
 		if (targetPeak >= persistentPeak)
 		{
 			persistentPeak = targetPeak;
 			lastPeakTimer.restart();
+			peakChanged(persistentPeak);
 		}
 		update();
 	}
 
 	if (persistentPeak > 0 && lastPeakTimer.elapsed() > 1500)
 	{
 		persistentPeak = qMax<float>(0, persistentPeak-0.05);
+		peakChanged(persistentPeak);
```

Comment:
We prefer to use the `emit` macro when emitting a signal.
```suggestion
			emit peakChanged(persistentPeak);
		}
		update();
	}

	if (persistentPeak > 0 && lastPeakTimer.elapsed() > 1500)
	{
		persistentPeak = qMax<float>(0, persistentPeak-0.05);
		emit peakChanged(persistentPeak);
```

Reply:
Fixed with commit aea000bc594.